src/parser/token.cpp

```diff
@@ -11,164 +11,163 @@
 #include "reference.h"
 #include "operation.h"
 
-
 // ---------------------------------------------------------------------------------------------------------------------
 // TODO:
 // 1. Match boolean tokens in any case (i.e. TRUE, FaLsE)
 
-Token* TokenConstructor(TokenType type, String content, int number)
+Token *TokenConstructor(TokenType type, String content, int number)
 {
-    Token* t = new Token { type, content, number };
```

Comment:
I'm not sure why this is happening but this is probably because of a delinter that you're using. Let's not include all of these changes with this PR. If needed, we can discuss what we'd like the repo conventions to be (I personally prefer, and this _is_ unorthodox, binding the `*` operator to the type definition).

Reply:
Is there a way that I can revert them that doesn't require me to manually change every line? I also looked through my extensions and can't find a linter that would be changing this.